core/src/main/scala-2/spire/std/double.scala

```diff
@@ -0,0 +1,148 @@
+/*
+ * **********************************************************************\
+ * * Project                                                              **
+ * *       ______  ______   __    ______    ____                          **
+ * *      / ____/ / __  /  / /   / __  /   / __/     (c) 2011-2021        **
+ * *     / /__   / /_/ /  / /   / /_/ /   / /_                            **
+ * *    /___  / / ____/  / /   / __  /   / __/   Erik Osheim, Tom Switzer **
+ * *   ____/ / / /      / /   / / | |   / /__                             **
+ * *  /_____/ /_/      /_/   /_/  |_|  /____/     All rights reserved.    **
+ * *                                                                      **
+ * *      Redistribution and use permitted under the MIT license.         **
+ * *                                                                      **
+ * \***********************************************************************
+ */
+
+package spire
+package std
+
+import spire.algebra.{Field, IsRational, NRoot, Order, Signed, Trig, TruncatedDivisionCRing}
+import spire.math.Rational
+import spire.util.Opt
+
+import java.lang.Math
+
+trait DoubleIsField extends Field[Double] {
+  override def minus(a: Double, b: Double): Double = a - b
+  def negate(a: Double): Double = -a
+  def one: Double = 1.0
```

Comment:
Why did you change this from `val` to `def` on 2.13? It broke bincompat.

Reply:
Wait, no, it was not this, sorry. Something else changed though on 2.13, let me find it ...

Reply:
Oh, probably left over from an earlier experiment and accidentally copied

Reply:
actually, these were `def` before:
https://github.com/typelevel/spire/blob/main/core/src/main/scala/spire/std/double.scala#L28